cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerConfigService.java

```diff
@@ -110,6 +110,20 @@ private void modifyKnoxAutorestart(ApiClient client, String clusterName, boolean
         }
     }
 
+    public Optional<String> getServiceNameIfExists(ApiClient client, String clusterName, String serviceType) {
+        try {
+            ServicesResourceApi servicesResourceApi = clouderaManagerApiFactory.getServicesResourceApi(client);
+            ApiServiceList serviceList = servicesResourceApi.readServices(clusterName, DataView.SUMMARY.name());
+            return serviceList.getItems().stream()
+                    .filter(service -> serviceType.equalsIgnoreCase(service.getType()))
+                    .map(ApiService::getName)
+                    .findFirst();
+        } catch (ApiException e) {
+            LOGGER.debug("Failed to get {} service name from Cloudera Manager.", serviceType, e);
```

Comment:
Could you please append the cluster name as well?

Reply:
Do you mean in the logs message?